cookbook/deployment/cluster/access_cloud_resources.py

```diff
@@ -1,8 +1,22 @@
 """
 Accessing Cloud Resources
---------------------------------------------
+-------------------------
```

Comment:
Do we need an intro paragraph here?

Reply:
Probably, though I'm not sure who's best positioned to write that. I think we should add a TODO comment here so we can keep track of what we need to fill in

Reply:
Got it.